joblib/test/test_numpy_pickle.py

```diff
@@ -598,6 +600,51 @@ def test_joblib_compression_formats():
                     os.remove(dump_filename)
 
 
+def _decompress_gzip(filename):
+    """Decompress a gzip file."""
+    with closing(gzip.GzipFile(filename + ".gz", "rb")) as fo:
```

Comment:
The implicit naming is not great. I'd rather have a function with less surprising signature:

```py
def _decompress_gzip(source_filename, target_filename):
    ...
```